city_scrapers/spiders/chi_northern_il_university.py

```diff
@@ -0,0 +1,111 @@
+import re
+from collections import defaultdict
+from datetime import datetime
+
+from city_scrapers_core.constants import BOARD, COMMITTEE, NOT_CLASSIFIED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiNorthernIlUniversitySpider(CityScrapersSpider):
+    name = "chi_northern_il_university"
+    agency = "Northeastern Illinois University"
+    timezone = "America/Chicago"
+    start_urls = ["https://www.neiu.edu/about/board-of-trustees/board-meeting-materials"]
+
+    def parse(self, response):
+        self.link_date_map = defaultdict(list)
+        for title in response.css(".field--name-field-generic-body h3"):
+            full_str = title.xpath(".//text()").extract()[0]
+            date_match = re.search(r"[a-z]{3,9} \d{1,2},? \d{4}", full_str, flags=re.I).group(0)
+            date_obj = datetime.strptime(re.sub(',', '', date_match), '%B %d %Y')
```

Comment:
If you're only replacing regular strings without regex formatting you can call `date_match.replace(',', '')` instead of using `re.sub`. This goes for a couple other spots in the spider as well.